src/ddmd/escape.d

```diff
@@ -37,6 +37,7 @@ import ddmd.arraytypes;
  * Print error messages when these are detected.
  * Params:
  *      sc = used to determine current function and module
+ *      fdc = function being called, `null` if called indirectly
```

Comment:
This kind of unrelated fixes, while welcome, should go in their own commit. If there is a bug showing up with the fix, and we end up reverting it, we'll loose this fix. It also make it easier to spot for reviewer, and is trivial to do using `git`.

Reply:
I understand your point, but at some level this just gets to be more pedantic than practical. It's just a comment.

Reply:
> but at some level this just gets to be more pedantic than practical. 

I think we are far away from this level. Proper commit messages, and proper commit separation help *a lot* on the long run. That's what we have here in Sociomantic, and while it sometimes gets annoying, I can't understate how useful it is when you have to track down information which is months or years old.

I don't think it should be a blocker for your PRs, but I just wanted to raise the point here.